src/m365/spo/commands/site/site-chrome-set.ts

```diff
@@ -0,0 +1,179 @@
+    telemetryProps.hideTitleInHeader = typeof args.options.hideTitleInHeader !== 'undefined';
+    telemetryProps.logoAlignment = typeof args.options.logoAlignment !== 'undefined';
+    telemetryProps.disableFooter = typeof args.options.disableFooter !== 'undefined';
+    telemetryProps.footerLayout = typeof args.options.footerLayout !== 'undefined';
+    telemetryProps.footerEmphasis = typeof args.options.footerEmphasis !== 'undefined';
+    return telemetryProps;
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+
+    const headerLayout = args.options.headerLayout ? HeaderLayout[args.options.headerLayout] : HeaderLayout.Standard;
+    const headerEmphasis = args.options.headerEmphasis ? Emphasis[args.options.headerEmphasis] : Emphasis.Lightest;
+    const logoAlignment = args.options.logoAlignment ? Alignment[args.options.logoAlignment] : Alignment.Left;
+    const footerLayout = args.options.footerLayout ? FooterLayout[args.options.footerLayout] : FooterLayout.Simple;
+    const footerEmphasis = args.options.footerEmphasis ? Emphasis[args.options.footerEmphasis] : Emphasis.Darkest;
+    const hideTitleInHeader = !!args.options.hideTitleInHeader;
+    const disableMegaMenu = typeof args.options.disableMegaMenu !== 'undefined' ? args.options.disableMegaMenu : false;
+    const disableFooter = typeof args.options.disableFooter !== 'undefined' ? args.options.disableFooter : false;
+
+    const requestOptions: any = {
+      url: `${args.options.url}/_api/web/SetChromeOptions`,
+      headers: {
+        accept: 'application/json;odata=nometadata'
+      },
+      data: {
+        headerLayout,
+        headerEmphasis,
+        megaMenuEnabled: !disableMegaMenu,
+        footerEnabled: !disableFooter,
+        footerLayout,
+        footerEmphasis: 3 - (footerEmphasis as number), // Footer is inverted
+        hideTitleInHeader,
+        logoAlignment
+      },
+      responseType: 'json'
+    };
+
+    request
+      .post(requestOptions)
+      .then((): void => cb(), (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-u, --url <url>'
+      },
+      {
+        option: '--headerLayout [headerLayout]',
+        autocomplete: ['Standard', 'Compact', 'Minimal', 'Extended']
+      },
+      {
+        option: '--headerEmphasis [headerEmphasis]',
+        autocomplete: ['Lightest', 'Light', 'Dark', 'Darkest']
+      },
+      {
+        option: '--logoAlignment [logoAlignment]',
+        autocomplete: ['Left', 'Center', 'Right']
+      },
+      {
+        option: '--footerLayout [footerLayout]',
+        autocomplete: ['Simple', 'Extended']
+      },
+      {
+        option: '--footerEmphasis [footerEmphasis]',
+        autocomplete: ['Lightest', 'Light', 'Dark', 'Darkest']
+      },
+      {
+        option: '--disableMegaMenu'
```

Comment:
Since this option is define as a flag, how would you enable the mega menu if it was disabled? Should we instead, specify it as an option with a value like `--disableMegaMenu [disableMegaMenu]`